docs/SOURCES.md

```diff
@@ -1,255 +1,972 @@
+| ▶️ **Current observation**     | Available: most recent real-time data from the nearest weather station           |
+| 📆 **Daily forecast**          | Up to 15 days                                                                    |
+| ⏱️ **Hourly forecast**         | Up to 16 days                                                                    |
+| 😶‍🌫️ **Air quality**         | Available                                                                        |
+| 🤧 **Pollen**                  | Available in Europe for: Alder, Birch, Grass, Mugwort, Olive, and Ragweed        |
+| ☔ **Precipitation nowcasting** | Available (works best in Europe at the moment)                                   |
+| ⚠️ **Alerts**                  | Not available: consider adding [Accuweather](#accuweather) as a secondary source |
+| 📊 **Normals**                 | Not available: consider adding [Accuweather](#accuweather) as a secondary source |
+| 🧭 **Reverse geocoding**       | Not available: will not show the name of device location                         |
+
+For the United States, [Forecast Advisor](https://www.forecastadvisor.com/) has temperature and precipitation 1-3 days accuracy comparison by city between the following sources: [Accuweather](#accuweather), [NWS](#national-weather-service), **Open-Meteo** and [Pirate Weather](#pirate-weather).
+
+<details><summary><h4>Details of available data from Open-Meteo</h4></summary>
+
+| Data                      | Available | Data              | Available |
+|---------------------------|-----------|-------------------|-----------|
+| Weather Condition         | ✅         | Pressure          | ✅         |
+| Temperature               | ✅         | UV Index          | ✅         |
+| Precipitation             | ✅         | Sunshine Duration | ✅         |
+| Precipitation Probability | ✅         | Sun &amp; Moon    | ✅         |
+| Precipitation Duration    | ❌         | Moon Phase        | ❌         |
+| Wind                      | ✅         | Cloud Cover       | ✅         |
+| Humidity                  | ✅         | Visibility        | ✅         |
+| Dew Point                 | ✅         | Ceiling           | ❌         |
+</details>
+
+### Accuweather
+**[Accuweather](https://www.accuweather.com/)** is a commercial weather data provider based in State College, Pennsylvania, United States.
+
+| Feature                        | Detail                                                                 |
+|--------------------------------|------------------------------------------------------------------------|
+| 🔐 **API key**                 | Optional: [Register](https://developer.accuweather.com/)               |
+| 🗺️ **Coverage**               | 🌐 Worldwide (some features may not be available for all locations)    |
+| ▶️ **Current observation**     | Available: most recent real-time data from the nearest weather station |
```

Comment:
Do we have confirmation they use station info and not forecast? At least I believe they don't support stations from all countries in the world, and fallback to forecast?

Reply:
Same as above re: sensible wording for this feature.